.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-on: [push, pull_request]
```

Comment:
Is there a reason for disabling workflows if a PR is created? I had created #415 because of contributors who do not have write permission to the repo.

Reply:
Ah, I see. Otherwise in our case it's running twice but then I'll put it back.